corehq/apps/export/templates/export/customize_export.html

```diff
@@ -7,452 +7,20 @@
-                    });
-                };
-                self.save.state = ko.observable('save');
-
-                self.save_no_preview = function() {
-                    var exportType = self.export_type();
-                    exportType = _(exportType).capitalize();
-                    var action = "Regular";
-                    if (self.presave()) {
-                        action = "Saved";
-                    }
-                    analytics.usage("Create Export", exportType, action);
-                    if (self.custom_export.default_format() === 'html') {
-                        analytics.usage("Create Export", exportType, "Excel Dashboard");
-                    }
-
-                    if (!self.custom_export._id || !self.custom_export._id()) {
-                        analytics.workflow("Clicked 'Create' in export edit page")
-                    }
-                    self.save(false);
-                };
-
-                self.save_preview = function() {
-                    self.save(true);
-                };
-
-                self.row_label_classes = function(row) {
-                    return (row === 'no data' || row === 'deleted') ? "label label-warning" : "label";
-                };
-
-                setTimeout(function () {
-                    _(self.table_configuration()).each(function (table) {
-                        if (!table.display()) {
-                            table.display(self.utils.tableHeader(table.index));
-                        }
-                        _(table.column_configuration()).each(function (column) {
-                            if (!column.display()) {
-                                var parsed = column._niceField;
-                                var prefixed_tags = ["case", "meta", "info", "server", "tag", "row"];
-                                var prefix = '';
-                                for (var i = 0; i < parsed.tags.length; i++) {
-                                    for (var j = 0; j < prefixed_tags.length; j++) {
-                                        if (parsed.tags[i] === prefixed_tags[j]) {
-                                            prefix = parsed.tags[i] + '.';
-                                            break;
-                                        }
-                                    }
-                                }
-                                var display = prefix + parsed.field;
-                                column.display(display);
-                            }
-                        });
-                    });
-                }, 0);
-
-                return self;
-            },
-            excludedTables: {
-                '#.#export_tag.#': 0,
-                '#.export_tag.#': 0,
-                '#.location_.#': 0,
-                '#.referrals.#': 0,
-                '#.xform_ids.#': 0
-            }
-        };
-
         $(function () {
+            var translations = {
+                forms: "{% trans 'Forms' %}",
```

Comment:
may want to add `escapejs` as precaution in case anyone translates these to something weird